gtfs-realtime-validator-lib/README.md

```diff
@@ -99,28 +98,25 @@ This will generate a JAR file in the `gtfs-realtime-validator-lib/target/` direc
 
 You can use the **gtfs-realtime-validator-lib** library in your own project as well.  
 
```

Comment:
I would add here:

>*NOTE: You need [to authenticate with GitHub](https://docs.github.com/en/packages/working-with-a-github-packages-registry/working-with-the-apache-maven-registry#authenticating-with-a-personal-access-token) to download the below artifacts*

Reply:
Done in [27e8037](https://github.com/MobilityData/gtfs-realtime-validator/pull/85/commits/27e80373b9cf3e17b208c5d89a8494211ad236ce)